stripevpaypal.md

```diff
@@ -0,0 +1,115 @@
+**Potential Redirects**  
+Depending on the setup, customers might be redirected to PayPal's website to complete the purchase, which can sometimes result in a disjointed user experience.
+
+**Disputes and Chargebacks**  
+Although PayPal offers seller protection, merchants can still face challenges with disputes and chargebacks, which can be time-consuming to resolve.  
+*Source:* [PayPal's Seller Protection Program](https://www.paypal.com/us/webapps/mpp/security/seller-protection-learn-more)
+
+## Pros and Cons of Integrating Stripe on Squarespace
+
+### Pros:
+
+**Ease of Integration**: 
+Both Squarespace and Stripe are designed for users who might not be technically savvy. They have made it simple to integrate Stripe as a payment method on a Squarespace website. [Squarespace Help](https://support.squarespace.com/hc/en-us/articles/206540797-Stripe)
+
+**Security**: 
+Stripe is PCI compliant, which means that it adheres to high security standards. When integrated with Squarespace, you can reassure your customers that their payment details are safe. [Stripe's Security](https://stripe.com/docs/security/stripe)
+
+**Real-time Payment Processing**: 
+Transactions happen in real-time, allowing businesses to quickly receive confirmations and proceed with order fulfillment. [Squarespace Help](https://support.squarespace.com/hc/en-us/articles/206540797-Stripe)
+
+**Currency Flexibility**: 
+Stripe supports over 135 currencies, allowing Squarespace merchants to sell internationally with ease. [Stripe Global](https://stripe.com/global)
+
+**Transparent Fees**: 
+Both Squarespace and Stripe have straightforward fee structures. There are no hidden charges, making it easier for businesses to manage their finances. [Stripe Pricing](https://stripe.com/pricing)
+
+---
+   
+### Cons:
+
+**Additional Costs**: 
+While Squarespace and Stripe have transparent fees, using both can add up. There's a fee from Squarespace for using their platform and a separate fee from Stripe for payment processing. [Squarespace Pricing](https://www.squarespace.com/pricing)
+
+**Limited to Stripe's Payment Methods**: 
+Although Stripe supports a wide range of payment methods, there might be some regional or niche payment methods it doesn't support. If a merchant's audience primarily uses one of these methods, they might miss out on sales. [Stripe Payment Methods](https://stripe.com/docs/payments/payment-methods/overview)
+
+**Potential for Chargebacks**: 
+Like with many other payment processors, Stripe merchants can face chargebacks. If a customer disputes a transaction, there is a possibility that the funds will be reversed, and the merchant might also incur a chargeback fee. [Stripe Documentation on Disputes](https://stripe.com/docs/disputes)
+
+**Dependency on Third-party Platforms**: 
+If either Squarespace or Stripe faces an outage or technical issue, it could it could prevent the busines from processing transactions. This creates a dependency on both platforms' reliability. 

```

Comment:
- delete second "it could"
- add second "s" on "business"

Reply:
- Removed second "it could"

- corrected business spelling